easybuild/easyconfigs/t/tmux/tmux-2.3.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'tmux'
+version = '2.3'
+
+homepage = 'http://tmux.sourceforge.net/'
+description = """tmux is a terminal multiplexer. It lets you switch easily 
+between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal."""
```

Comment:
@foxik0070 line too long, should be <= 120 characters

please use description like we have in https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/t/tmux/tmux-2.3-GCC-5.4.0-2.26.eb#L7 ?